Assets/Scripts/UI/Roots/UIRoot.cs

```diff
@@ -28,7 +28,7 @@ public void Construct(UIBuilder uiBuilder)
                 return default;
             }
             
-            if(_viewStack.Any())
+            if (_viewStack.Any())
```

Comment:
This changes not requaried for you task pls. revert this file

Reply:
Reverted